cfme/tests/cloud/test_cloud_events.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""This module tests only cloud specific events"""
+import pytest
+import re
+
+from cfme.common.vm import VM
+from utils import testgen
+from utils.events import EventBuilder
+from utils.generators import random_vm_name
+
+
+pytest_generate_tests = testgen.generate(testgen.cloud_providers, scope="module")
```

Comment:
I tried this but azure wasn't picked up in this case. 
azure tests can be run only if --use-provider=azure is used.

Actually, I'm planning to put all cloud provider event tests here.
So, that's a reason for using cloud_providers in this file.